pkg/cmd/volume/list.go

```diff
@@ -48,18 +48,49 @@ func List(options *types.VolumeListCommandOptions, stdout io.Writer) error {
 		logrus.Warn("cannot use --size and --quiet together, ignoring --size")
 		options.Size = false
 	}
-	labelFilterFuncs, nameFilterFuncs, sizeFilterFuncs, isFilter, err := getVolumeFilterFuncs(options.Filters)
-	if err != nil {
-		return err
-	}
-	if len(sizeFilterFuncs) > 0 && options.Quiet {
+	sizeFilter := hasSizeFilter(options.Filters)
+	if sizeFilter && options.Quiet {
 		logrus.Warn("cannot use --filter=size and --quiet together, ignoring --filter=size")
-		sizeFilterFuncs = nil
+		options.Filters = removeSizeFilters(options.Filters)
```

Comment:
not related to this PR, but curious why this two flags conflict (`--filter=size` is for functionality/which volumes to `list`, while `--quiet` is for formatting/how to print). can we remove this restriction now? @AkihiroSuda

Reply:
Maybe yes, but should be another PR